### PR TITLE
Jetpack Disconnect: Add Icon to 'Diagnose a connection problem'

### PR DESCRIPTION
**Pull request by ockham**
As suggested by @tyxla (p7rd6c-15q-p2#comment-1768):

> I think it would be more consistent if we have an icon to the left of the “Diagnose a connection problem link” text. There are many icons that might work here – for example offline, notice/notice-outline, bug, etc.

_notice_/_notice-outline_ don't seem right, I think they're really meant to notify the user of something that's wrong. _Bug_ doesn't feel quite right either, so I chose _offline_ which I think is the best fit.

Before:

image

After:

image

Ignore the misalignment for now, I'll address that in a separate PR.

To test:
* Have a look at `/settings/disconnect-site/:site`.

### CODE REVIEW

File: client/my-sites/site-settings/disconnect-site/troubleshoot.jsx [hunk 1]
```diff
@@ -5,6 +5,7 @@
 import React from 'react';
 import { connect } from 'react-redux';
 import { localize } from 'i18n-calypso';
+import Gridicon from 'gridicons';
```
Comment (tyxla) [inline client/my-sites/site-settings/disconnect-site/troubleshoot.jsx]: (behold the king of the nits)

ABC police 🔤  - letters come before curly braces, so the `Gridicon` import should go below the `React` one (or above it, depending on how 🔤 -fanatic you are 😉 ).

Comment (ockham) [reply client/my-sites/site-settings/disconnect-site/troubleshoot.jsx]: 2bbef65

**Comment by folletto** [comment]
I wasn't sure about using the `offline` icon for this... but since it's for "connection problems", it might actually make a lot of sense.
I'm ok with that if we want to use it.

Otherwise, I think that `bug` or `notice-outline` would be good choices too.

**Comment by tyxla** [comment]
Thanks @folletto 🙇

**Comment by rickybanister** [comment]
The offline icon works well-enough. There's some kind of icon alignment issue though. 

The icon appears too high and also the width or spacing or something is off:

image

**Comment by ockham** [comment]
> There's some kind of icon alignment issue though.

> The icon appears too high and also the width or spacing or something is off: [...]

Yep, on my radar, see PR desc :slightly_smiling_face: 

> Ignore the misalignment for now, I'll address that in a separate PR.

**Comment by tyxla** [comment]
https://github.com/Automattic/wp-calypso/pull/19327/commits/8e44e4623449c95ba3d7adac83b159535fd075a0 addresses the alignment issues - it required both vertical and horizontal touch.

**Comment by tyxla** [comment]
#19346 tackles the icon alignment issue and takes care of consistency with other occurrences of `Gridicon` within `LoggedOutFormLinkItem`.

**Pull request merged**